src/components/Create.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Card } from "antd";
+
+class Create extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: "",
+      body: ""
+    };
+  }
+
+  handleChangeTitle = event => {
+    this.setState({
+      title: event.target.value
+    });
+  };
+
+  handleChangeBody = event => {
+    this.setState({
+      body: event.target.value
+    });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    const data = {
+      title: this.state.title,
+      body: this.state.body
+    };
+
+    axios.post(`http://5dccfe88d795470014e4ca93.mockapi.io/test`, data)
+      .then(res => {
+        console.log(res.data);
```

Comment:
Sau khi create thành công thì nên trả về trang list. this.props.history.push('/')